matrix_client/client.py

```diff
@@ -127,6 +127,10 @@ def start_listener_thread(self, timeout=30000):
             e = sys.exc_info()[0]
             print("Error: unable to start thread. " + str(e))
 
+    def upload(self,content,content_type):
+        response = self.api.media_upload(content,content_type)
+        return response["content_uri"]
```

Comment:
Would it be worth having a separate exception for when the library gets unexpected data back from a successful API request?